test/github-issues/8667/issue-8667.ts

```diff
@@ -0,0 +1,85 @@
+import "reflect-metadata"
+import { expect } from "chai"
+import { DataSource } from "../../../src"
+import { Post } from "./entity/Post"
+import {
+    createTestingConnections,
+    reloadTestingDatabases,
+    closeTestingConnections,
+} from "../../utils/test-utils"
+
+describe("github issues > #8667 Support RETURNING clause in SQLite", () => {
+    let connections: DataSource[]
+    let post: Post
+
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [Post],
+            schemaCreate: true,
+            dropSchema: true,
+            enabledDrivers: ["sqlite", "better-sqlite3", "sqljs"],
+        })
+
+        post = new Post()
+        post.title = "title"
+        post.text = "text"
+    })
+
+    beforeEach(() => reloadTestingDatabases(connections))
+    after(() => closeTestingConnections(connections))
+
+    it("should return all columns via QueryRunner", async () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                await connection
+                    .createQueryBuilder()
+                    .insert()
+                    .into(Post)
+                    .values(post)
+                    .execute()
+
+                const queryRunner = connection.createQueryRunner()
+
+                const { tableName } = connection.getMetadata(Post)
+
+                await queryRunner.connect()
+
+                const res = await queryRunner.query(
+                    `UPDATE ${tableName} SET title = 'updated title value' RETURNING *`,
+                )
+
+                expect(res).to.be.deep.equal([
+                    { id: 1, title: "updated title value", text: "text" },
+                ])
+            }),
+        ))
+
+    it("should return user requested columns via QueryBuilder", () =>
```

Comment:
Would be nice if you could extract this test to `test/functional/query-builder/returning/query-builder-returning-columns.ts`.

Since this is almost identical to `test/github-issues/7100/issue-7100.ts`, they could be combined (7100 can be copied, sqlite drivers added, and the missing assertion after insert can be added like in this test).